app/views/ncr/work_orders/form.html.erb

```diff
@@ -72,10 +72,13 @@
     </div>
     <div class="form-group">
       <%= field_set_tag "Amount", class: "required" do %>
-        <%= f.label :amount, "Dollar Amount", :class => 'sr-only'  %>
+        <%= f.label :amount, "Dollar Amount", class: 'sr-only' %>
         <div class="input-group">
           <div class="input-group-addon">$</div>
-          <%= f.number_field :amount, class: 'form-control', step: 0.01 %>
+          <%= f.number_field :amount, class: 'form-control', step: 0.01,
+                data: { toggle: 'popover', trigger: 'focus', html: true,
+                  title: 'Transaction Amount Thresholds', placement: 'top',
+                  content: "$3,000 for supplies<br>$2,500 for services<br>$2,000 for construction" } %>
```

Comment:
maybe we can throw this text into a helper? Or one of your BFF decorators? Not sure if we've been extracting form or view logic in either of those much up to this point, but it's always nice to have simpler views